### DISCUSSION TITLE
Summary field for multiple page forms

### DISCUSSION THREAD

**Discussion by FrDH** [solspace/craft-freeform#933] [Questions & Answers] [answered]
Couldn't find it in the docs, but we want something like this:
https://formidableforms.com/knowledgebase/summary-review-before-submit/

So the visitor can review what he is going to submit, before actually submittin.

For now: Can I show the filled in values in a template before a form is actually submitted?
Basically like this but before having submitted:
https://docs.solspace.com/craft/freeform/v2/guides/submission-data-success/#show-submission-data-on-success-page

**Comment by kjmartens** [comment, marked as answer]
Hi @FrDH,

We actually have a guide for this:
https://docs.solspace.com/craft/freeform/v4/guides/displaying-submission-data-from-previous-pages/

You will basically need to use a multipage form and then set up an HTML field with Twig on the last page to populate the data from previous pages.

Please let me know if you have any further questions. 🙂

**Comment by FrDH** [comment]
Thanks, that is exactly what I was looking for 👍
